privacyidea/static/components/config/views/config.smtp.edit.html

```diff
@@ -16,6 +16,8 @@
             <input name="identifier" id="identifier" class="form-control"
                    ng-model="params.identifier" required
                    placeholder="mySmtpServer"
+                   pattern="{{ inputNamePatterns.simple.pattern }}"
+                   title="{{ inputNamePatterns.simple.title }}"
```

Comment:
*argh*! :-/
We have an endpoint ``/send_test_email``.
Should we add a pattern here like with test? Or should we rename the endpoint?

Reply:
Added a pattern in 84b8a8a0